### DISCUSSION TITLE
Re-evaluate tuning strategy, now and going forward.

### DISCUSSION THREAD

**Discussion by heyitsbench** [chromiecraft/chromiecraft#7233] [Suggestions]
As bugs get resolved in instance content, tuning needs to be adjusted to match. This has not been more apparent than with Archimonde. First Archimonde was challenging, but beatable. Bug gets fixed, he gets noticeably easier. Bug gets fixed, he gets substantially harder. Bug gets fixed, little easier. Bug gets fixed, a lot more difficult.

Quite frankly, in my opinion (and many others), the tuning just shouldn't be as high as it is. Many players joined this server expecting it to be casual-friendly and, well, relatively easily cleared week one. The past few raid releases have been a slap in the face to almost every player expecting that. Players are being met with bosses that very much feel like they require BiS geared members, favorable compositions, and lady luck on their side in order to have a chance to consistently defeat. It should come as no surprise that this is genuinely killing raid groups. The constant pendulum swing grinds players down, and with 25 to a raid group, it doesn't take many dominoes to take a group out of the playerbase.

I understand tuning is a challenge, I understand it can be a fine balance, but if tuning can't change quickly (as in week to week), then it cannot be tuned so high where any severe impact bug being corrected could significantly change the way a fight is played.

Hand in hand with this, Archimonde needs retuning ASAP. He's had so many bugs fixed that make the fight more difficult that the already rather high tuning relegates him to a position where he only frustrates players in an already frustrating raid.

**Comment by Krateozshael** [comment]
Fully agree on this, nothing more to be said

**Comment by quigiboo** [comment]
Agreed, Archimonde is in a terrible state right now and I've personally witnessed this actively damaging the server population.

Please remember this is supposed to be a casual friendly server, things don't have to be incredibly over tuned to be able to properly test mechanics.

**Comment by Undeadhunter** [comment]
Server is not fun anymore, which is the worst thing it can do out of everything.

**Reply by bigdeekpumper** [reply]
Fun is a subjective measure but the most fun i had on this server when it comes to raiding was the period between vanilla to T4. Lots of different pugs happening every day, you could easily raid on alts both because the gear requirement wasn't very high and the raid mechanics/tuning weren't as pug unfriendly as they are now. The gap between a "good" group and a "bad" group was mostly how fast each cleared the raids. 

Starting from SSC, "bad" groups struggle hard and couldn't kill some bosses (even after the nerfs) to the point that you had to wonder if it was even worth it to join them, because you were going to have a guaranteed bad time. You see, progression is fun if you're in a guild, but progressing as a pug isn't fun at all, specially when each week it was a new set of people.

"Just join a guild then"
My main character always had a guild, that isn't the issue, however, after SSC released I stopped raiding with my alts entirely, it was just painful and not fun at all. And I couldn't fit my schedule to raid on the so called "good" pug groups. In contrast with the old times, I could pug almost any day of the week, at least from what I remember. 
If people stop raiding on their alts you simply destroy the "pug culture" that was my favorite thing about the server, and i'm sure i'm not alone on this. I used to even host pugs, but after SSC I wouldn't dare, I care about my well being and sanity. 

Then Hyjal releases and the tuning makes one of the worst raids in WoW history even worse than I could have imagined. All this pain just for it to be nerfed to the ground in 2 months, like they've done with SSC/TK. I prefer playing easy raids, but if you're going the hard raids route these nerfs make no sense and remove any motivation to raid for me. (Why suffer now if in X weeks Hyjal will be easier than a heroic dungeon?)

Why even bother!?

**Comment by joesill** [comment]
But don't you know that pugs can clear it? Its just a matter of learning how to play!

It doesn't matter that the current attitude wrt tuning brought the server to the glorious state of 200 online players during prime time.

The only thing that matters is that pugs can clear it.

**Reply by shaszun** [reply]
I hope this is a sarcasm post.

**Reply by joesill** [reply]
maybe it is, maybe it isn't

"pugs can clear it" is an argument widely used in discussions about tuning in the CC discord, and it seems to hold weight, as the tuning remains unchanged

and the current playerbase seems optimal, as the gms do nothing to retain current players  or draw new players in
